src/Pages/Adopt.jsx

```diff
@@ -217,247 +288,406 @@ export const Adopt = () => {
-                variant="outlined"
-              />
-              <TextField
-                id="veterinary-clinic-address"
-                label="Veterinarian Clinic Address"
-                variant="outlined"
-              />
-              <TextField
-                id="veterinary-clinic-phone"
-                label="Veterinarian Clinic Phone Number"
-                variant="outlined"
-              />
-              <FormLabel
-                id="residence-type"
-                sx={{ fontWeight: "bold" }}
-              >
-                Residence
-              </FormLabel>
-              <RadioGroup
-                aria-labelledby="residence-type"
-                defaultValue="single-family-house"
-                name="residence-type-radio-buttons-group"
-              >
-                <FormControlLabel
-                  value="single-family-house"
-                  control={<Radio />}
-                  label="Single Family House"
+                <TextField
+                  id="household-ages"
+                  label="What are the ages of everyone in the household?"
+                  variant="outlined"
+                  {...register("ages_in_household", {
+                    required: "Household members' ages are required",
+                  })}
+                  error={!!errors.ages_in_household?.message}
+                  helperText={errors.ages_in_household?.message}
                 />
-                <FormControlLabel
-                  value="mobile-home"
-                  control={<Radio />}
-                  label="Mobile Home"
+                <TextField
+                  id="children-experience-with-birds"
+                  label="Are there children in the household? What is their experience with birds?"
+                  variant="outlined"
+                  multiline
+                  minRows={4}
+                  {...register("children_in_house", {
+                    required: "Children's experience with birds is required",
+                  })}
+                  error={!!errors.children_in_house?.message}
+                  helperText={errors.children_in_house?.message}
                 />
-                <FormControlLabel
-                  value="condo"
-                  control={<Radio />}
-                  label="Condo"
+                <FormLabel
+                  id="children-in-future"
+                  sx={{ fontWeight: "bold" }}
+                >
+                  Do you currently have other birds in the household?
+                </FormLabel>
+                <Controller
+                  control={control}
+                  name="have_other_birds"
+                  render={({ field: { onChange, value, name } }) => (
+                    <RadioGroup
+                      aria-labelledby="other-birds-in-household-bool"
+                      value={value}
+                      name={name}
+                      onChange={(evt) => {
```

Comment:
This is a tiny nitpick (sorry! 😁), but in my experience, the event parameter in an onChange is usually named either `e` or `event`. I personally prefer either of those to `evt`, which is not as common.

Reply:
Sounds good! Thanks for the feedback. I personally like to differentiate between an error and an event, which can both often be passed into arrow functions in similar settings. For that reason, I don't like just "e" but "event" is good for me.